packages/styles/src/components/stepper.scss

```diff
@@ -104,35 +98,38 @@
   }
 }
 
+.stepper-interactive {
```

Comment:
This new class seems like a breaking change as projects need to add it to their steppers to not lose interactivity on completed steps...

Can't we just say `a.stepper-link` are interactive and `span.stepper-link` aren't?
This probably implies a larger refactoring to attached the `::before` to the `stepper-link` instead of the `stepper-item`.